src/benchmarks/gc/scenarios/CrankConfiguration.yaml

```diff
@@ -28,9 +28,9 @@ jobs:
       pohfi: 0
       allocType: reference
       testKind: time
-    arguments: "-tc {{ tc }} -tagb {{ tagb }} -tlgb {{ tlgb }} -lohar {{ lohar }} -lohsr {{ lohsr }} -sohsi {{ sohsi }} -lohsi {{ lohsi }} -sohpi {{ sohpi }} -lohpi {{ lohpi }} -sohfi {{ sohfi }} -lohfi {{ lohfi }} -allocType {{ allocType }} -testKind {{ testKind }}"
+    arguments: "-tc {{ cores | minus : 2 | times: 2 }} -tagb {{ tagb }} -tlgb {{ tlgb }} -lohar {{ lohar }} -lohsr {{ lohsr }} -sohsi {{ sohsi }} -lohsi {{ lohsi }} -sohpi {{ sohpi }} -lohpi {{ lohpi }} -sohfi {{ sohfi }} -lohfi {{ lohfi }} -allocType {{ allocType }} -testKind {{ testKind }}"
```

Comment:
This should really be `{{ tc }}` and you should update the default value of `tc` instead

Reply:
Otherwise you will never be able to set anything else